lib/iu_metadata/marc_record.rb

```diff
@@ -209,10 +210,12 @@ def title(include_initial_article = true)
 
     def subject
       # Broken: name puctuation won't come out correctly
-      formatted_fields_as_array([
+      formatted_fields_as_array
+      ([
```

Comment:
I think this being on a new line would cause the method to be called without any arguments.